### ISSUE TITLE
installation of 1.3.18 fails on Windows

### ISSUE THREAD

**Issue by iev6** [imdone/imdone-atom#66]
npm install error - MSB3428: Could not load the Visual C++ component “VCBuild.exe”

**Comment by jronallo** [comment]
@iev6 Is there any other output that you can share? Can you provide the full error message that Atom shows? I'm wondering if this is in any way related to #65 or not.

**Comment by iev6** [comment]
Enclosed below is the log : 

`````````

> runas@3.1.1 install C:\Users\girid\AppData\Local\Temp\apm-install-dir-116010-2292-2pr5l\node_modules\imdone-atom\node_modules\pathwatcher\node_modules\runas
> node-gyp rebuild


C:\Users\girid\AppData\Local\Temp\apm-install-dir-116010-2292-2pr5l\node_modules\imdone-atom\node_modules\pathwatcher\node_modules\runas>if not defined npm_config_node_gyp (node "C:\Users\girid\AppData\Local\atom\app-1.3.3\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node  rebuild ) 
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
C:\Users\girid\AppData\Local\Temp\apm-install-dir-116010-2292-2pr5l\node_modules\imdone-atom\node_modules\pathwatcher\node_modules\runas\build\runas.vcxproj(20,3): error MSB4019: The imported project "C:\Microsoft.Cpp.Default.props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.

gypnpm WARN optional dep failed, continuing fsevents@1.0.6
npm ERR! Windows_NT 6.2.9200
npm ERR! argv "C:\\Users\\girid\\AppData\\Local\\atom\\app-1.3.3\\resources\\app\\apm\\bin\\node.exe" "C:\\Users\\girid\\AppData\\Local\\atom\\app-1.3.3\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\girid\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\girid\\.atom\\.apmrc" "install" "C:\\Users\\girid\\AppData\\Local\\Temp\\d-116010-2292-1azxe5m\\package.tgz" "--target=0.34.5" "--arch=ia32" "--msvs_version=2013"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3
npm ERR! code ELIFECYCLE

npm ERR! runas@3.1.1 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the runas@3.1.1 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the runas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls runas
npm ERR! There is likely additional logging output above.

`````

**Comment by iev6** [comment]
This is a node.js error that appears in Windows. Only workaround seems to be reinstalling VC++ libraries.

**Comment by piascikj** [comment]
Should be fixed with 1.3.19
